IDR/notebook/nbconverted/0.get_ids.py

```diff
@@ -1,59 +1,40 @@
-#!/usr/bin/env python
-# coding: utf-8
-
-# # Obtain details for all screens and projects on Image Data Resource
-# 
-# https://idr.openmicroscopy.org/
-
-# In[1]:
-
-
 import pathlib
 import requests
 import pandas as pd
 
 
-# In[2]:
-
-
 def get_id(json):
     """Clean json details in preparation for IDR screen detail storage
-    
+
     Parameters
     ----------
     json: dict
         One IDR study, with keys storing important pieces of information about the study
-        
+
     Returns
     -------
     Tuple of ID, name, project title, description, and kind of study.
     """
     id_ = json["@id"]
     name_ = json["Name"]
     details_ = json["Description"]
-    
+
     if "Screen Description" in details_:
         split_detail = "Screen"
     elif "Study Description" in details_:
         split_detail = "Study"
     elif "Experiment Description" in details_:
         split_detail = "Experiment"
-    
+
     title_, description_ = details_.split(f"{split_detail} Description\n")
     title_ = title_.replace("Publication Title\n", "").replace("\n", "")
     description_ = description_.replace("\n", "")
 
     return (id_, name_, title_, description_, split_detail)
 
 
-# In[3]:
-
-
-output_dir = pathlib.Path("data")
-
-
-# In[4]:
-
+output_dir = pathlib.Path(
```

Comment:
Please use relative paths (instead of absolute paths).

Absolute paths are not reproducible (_most_ times) while relative paths are.

Also, the relative path should work _from the perspective of the `.ipynb` file, and not the nbconverted file_

Reply:
Absolute paths were changed to relative paths for both `.ipynb` and `.py` files.

Added in c509a494b51f1ca39a45777a25da8b707f28afac, 18899481ceeb0e5ba507f8c3447157288a142b8c, 9baa675ef256d827c020d66cd65ecd470b1c01dc, 6f50a07e5626509845c96a71cf3f9caaabd96ac3, and cca6823ea3d63ccf86200b07607b751ab3be8829.

Reply:
FYI, looks like there is an error in one of the `.ipynb` commits (not sure if this is fixed subsequently).

Also (and we'll learn this more once you get acclimated with our processes), but the comment:

> Please use relative paths (instead of absolute paths)

_is a sufficient for one single commit_. Your changes here are accurate (to the extent they don't break the code!), but they most likely belong in a single commit.

Reply:
Yes I wanted to do that in a single commit, but for some reason I forgot that you can add multiple files to a single commit in the terminal when I was doing this. My bad!